packages/gestalt/src/Icon/icons/index.js

```diff
@@ -44,8 +44,11 @@ import heartBroken from './heart-broken.svg';
 import link from './link.svg';
 import location from './location.svg';
 import lock from './lock.svg';
+import maximize from './maximize.svg';
```

Comment:
@pfarejowicz did you confirm that these icons come from PDS spec?

Reply:
Yes, I got all the new icons from the experimental folder that @jaymarsh maintains

Reply:
@jackhsu978 good catch with the maximize icon pointing in the different direction, I didn't even notice. I took that maximize icon straight from the PDS icons folder so my guess is that the flip was intentional, we can talk with @jaymarsh as to why the arrows point in the opposite direction now